src/tabs/AOCTab/SolutionView.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Day1Solution from './AnswerScripts/Day1Script';
+import Day2Solution from './AnswerScripts/Day2Script';
+import Day3Solution from './AnswerScripts/Day3Script';
+
+// When tab clicked, key of file gets inserted in as a prop
+// This component should take in a string for file name.
+// Inserts string into function that reads in script to a tag
+
+
+// const answer_query = { day: i - 1, file: file_name[i]}
+// Does it matter which way I do this? 
+
+// const leos_scripts = [
+//   Day1Solution,
+//   Day2Solution,
+//   Day3Solution,
+// ];
+
+// I want to keep solutions as python files for now.  I can learn 
+// how to convert them to JS after they've been set up and displayed properly
+const scripts = [
+  Day1Solution,
+  Day2Solution,
+  Day3Solution,
+];
+
+// When specific button is clicked, pull up specific solution
+// Link array to components that display each solution
+const SolutionView = (props) => {
+  // Should I open the file here?
+// Insert string into function that reads in script to a tag
+  return(    
+    <div>
+      {/* <p>Day: { props.day }</p> */}
+      <p>{ scripts[props.day - 1]} </p>
+      {/* Figure out how to display script from the file */}
+      <script>open({ scripts[props.day - 1] })</script>

```

Comment:
We should also put this into a function, but.... does this work?? For some reason i can't see it running, but I could be totally wrong.

Reply:
It doesn't work hahaha I just wanted to get the idea down.  I was thinking I should have the file opening logic right there, but it sounds like I should open each in each component, then display the components when each is clicked

Reply:
Then yeah this makes sense, it could go here for sure